From 9b280449f99b5273b52d88574c7358a874f825a3 Mon Sep 17 00:00:00 2001
From: Ting Liu <ting.liu@nxp.com>
Date: Mon, 17 Jun 2019 09:27:53 +0200
Subject: [PATCH] staging: fsl_ppfe/eth: prefix header search paths with
 $(srctree)/

Currently, the rules for configuring search paths in Kbuild have
changed: https://lkml.org/lkml/2019/5/13/37

This will lead the below error:

fatal error: pfe/pfe.h: No such file or directory

Fix it by adding $(srctree)/ prefix to the search paths.

Signed-off-by: Ting Liu <ting.liu@nxp.com>
---
 drivers/staging/fsl_ppfe/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/fsl_ppfe/Makefile
+++ b/drivers/staging/fsl_ppfe/Makefile
@@ -2,7 +2,7 @@
 # Makefile for Freesecale PPFE driver
 #
 
-ccflags-y +=  -I$(src)/include  -I$(src)
+ccflags-y +=  -I $(srctree)/$(src)/include  -I $(srctree)/$(src)
 
 obj-m += pfe.o
 
